integrations/git_test.go

```diff
@@ -5,9 +5,9 @@
 package integrations
 
 import (
-	"crypto/rand"
 	"fmt"
 	"io/ioutil"
+	"math/rand"
```

Comment:
You've changed from crypto/rand to math/rand here. What this deliberate?

Reply:
No, I've changed it back.